K2Bridge/JsonConverters/DateHistogramAggregationConverter.cs

```diff
@@ -23,10 +23,14 @@ internal class DateHistogramAggregationConverter : ReadOnlyJsonConverter
         {
             var jo = JObject.Load(reader);
 
+            var fixedInterval = jo["fixed_interval"];
+            var calendarInterval = jo["calendar_interval"];
+            var interval =
+                fixedInterval != null ? (string)fixedInterval : (string)calendarInterval;
             var obj = new DateHistogramAggregation
             {
                 FieldName = (string)jo["field"],
-                Interval = (string)jo["interval"],
+                Interval = interval,
```

Comment:
explain: Interval can be 1M, month, 1d, day, week, 2d, etc
but the key can be either fixed_interval of calendar_interval, so the logic in this file gets the correct value from either key